### DISCUSSION TITLE
cannot create a package with monai-deploy with version 0.6.0.

### DISCUSSION THREAD

**Discussion by mpsampat** [Project-MONAI/monai-deploy-app-sdk#463] [General]
1. I follow the instructions for simple_imaging_app for monai-deploy-app-sdk (0.6.0). 
2. I am on a VM on vertex AI (cpu only). 
3. the first one works: 
4. python examples/apps/simple_imaging_app/app.py -i examples/apps/simple_imaging_app/input/brain_mr_input.jpg -o output

5. the second one fails (the error below)
6.  monai-deploy package examples/apps/simple_imaging_app -c examples/apps/simple_imaging_app/app.yaml -t simple_app:latest --platform x64-workstation -l DEBUG

Do i need to have a GPU to package an app ? I know this was not the case before 0.6.0. 

Thanks
Mehul 

#21 [ 2/21] RUN mkdir -p /etc/holoscan/         && mkdir -p /opt/holoscan/         && mkdir -p /var/holoscan         && mkdir -p /opt/holoscan/app         && mkdir -p /var/holoscan/input         && mkdir -p /var/holoscan/output
#21 CANCELED
------
 > [16/21] COPY ./None /tmp/None:
------
WARNING: local cache import at /home/jupyter/.holoscan_build_cache not found due to err: could not read /home/jupyter/.holoscan_build_cache/index.json: open /home/jupyter/.holoscan_build_cache/index.json: no such file or directory
Dockerfile:78
--------------------
  76 |     
  77 |     # Copy user-specified MONAI Deploy SDK file
  78 | >>> COPY ./None /tmp/None
  79 |     RUN pip install /tmp/None
  80 |     
--------------------
error: failed to solve: failed to compute cache key: failed to calculate checksum of ref mke4lzitgojyb5voax49slol1::zr4jj9ad6e3y5u50bkyr5e3u0: "/None": not found
[2023-10-12 01:36:10,128] [INFO] (packager) - Build Summary:

Platform: x64-workstation/dgpu
    Status: Failure
    Error:  Error building image: see Docker output for additional details.

**Comment by MMelQin** [comment]
@mpsampat Sorry about this error. It is a known issue in the Packager, [tracked ](https://github.com/Project-MONAI/monai-deploy-app-sdk/issues/453)also as an one in App SDK with workaround. But within Jupyter notebook, the simpler workaround can be used, i.e. modify the package command to set the option to supply the MONAI Deploy App SDK Wheel, which can be downloaded from pypi.org.

**Reply by ChiariRay** [reply]
@MMelQin Hi, I have the same issue both in jupyter and CLI. Is there any solution to deal with it?

**Comment by nitin271080** [comment]
when using monai-depoy package command 
| # Copy user-specified MONAI Deploy SDK file
78 | >>> COPY ./None /tmp/None
79 | RUN pip install /tmp/None
this error generate 

use | # Copy user-specified MONAI Deploy SDK file
78 | >>> COPY ./None /tmp/None
79 | RUN pip install /tmp/None

use sub option ----monai-deploy-sdk-file  monai_deploy_app_sdk-0.6.0-py3-none-any.whl with monai-deploy package command


like 

monai-deploy package simple_imaging_app -c simple_imaging_app/app.yaml -t {tag_prefix}:1.0 --platform x64-workstation --monai-deploy-sdk-file  monai_deploy_app_sdk-0.6.0-py3-none-any.whl -l DEBUG

i will work
